Pipes - Iuliia - solar_system.rb

```diff
@@ -0,0 +1,140 @@
+class Planet
+  attr_accessor :name, :mass, :distance, :diameter, :orbit
+
+  def initialize(name, mass, distance, diameter, orbit)
+    @name = name
+    @mass = mass.to_f
+    @distance = distance.to_f
+    @diameter = diameter.to_f
+    @orbit = orbit.to_f
+  end
+
+  def info
+    return "Planet: #{@name}\nMass: #{@mass}\nDistance: #{@distance}\nDiameter: #{@diameter}\nOrbit: #{@orbit}"
+  end
+
+end #end of class Planet
+
+class SolarSystem
+
+  attr_accessor :planets, :age
+
+  def initialize(initial_planets, age)
+    @planets = initial_planets
+    @age = age
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def planet_list
+    result = ""
+    @planets.each do |planet|
+      result +=  "#{planet.info}\n\n"
```

Comment:
Good use of the existing `info` method on your `Planet` class. That makes this code much simpler.